In [13]:
import random
import math
node_num = 10
gw_x = 500.00
gw_y = 500.00
print("**.loRaGW[0].**.initialX = " + str(gw_x) + "m")
print("**.loRaGW[0].**.initialY = " + str(gw_y) + "m")
locations = []
distances = []
for i in range(node_num):
    x, y = round(gw_x + random.uniform(-100,100),2), round(gw_y + random.uniform(-100,100),2)
    locations.append([x,y])
    distances.append(math.sqrt((gw_x-x)**2 + (gw_y-y)**2))
    print("**.loRaNodes[" + str(i) + "].**.initialX = " + str(x) + "m")
    print("**.loRaNodes[" + str(i) + "].**.initialY = " + str(y) + "m")
print(locations)
print(distances)

**.loRaGW[0].**.initialX = 500.0m
**.loRaGW[0].**.initialY = 500.0m
**.loRaNodes[0].**.initialX = 432.51m
**.loRaNodes[0].**.initialY = 481.56m
**.loRaNodes[1].**.initialX = 545.87m
**.loRaNodes[1].**.initialY = 435.16m
**.loRaNodes[2].**.initialX = 476.36m
**.loRaNodes[2].**.initialY = 518.18m
**.loRaNodes[3].**.initialX = 484.83m
**.loRaNodes[3].**.initialY = 468.03m
**.loRaNodes[4].**.initialX = 547.28m
**.loRaNodes[4].**.initialY = 418.85m
**.loRaNodes[5].**.initialX = 442.3m
**.loRaNodes[5].**.initialY = 494.69m
**.loRaNodes[6].**.initialX = 528.67m
**.loRaNodes[6].**.initialY = 541.73m
**.loRaNodes[7].**.initialX = 551.7m
**.loRaNodes[7].**.initialY = 515.53m
**.loRaNodes[8].**.initialX = 425.33m
**.loRaNodes[8].**.initialY = 580.96m
**.loRaNodes[9].**.initialX = 501.35m
**.loRaNodes[9].**.initialY = 589.68m
[[432.51, 481.56], [545.87, 435.16], [476.36, 518.18], [484.83, 468.03], [547.28, 418.85], [442.3, 494.69], [528.67, 541.73], [551.7, 515.53], [425.33, 580.96], [501.35, 589.

**.loRaGW[0].**.initialX = 500.0m
**.loRaGW[0].**.initialY = 500.0m
**.loRaNodes[0].**.initialX = 432.51m
**.loRaNodes[0].**.initialY = 481.56m
**.loRaNodes[1].**.initialX = 545.87m
**.loRaNodes[1].**.initialY = 435.16m
**.loRaNodes[2].**.initialX = 476.36m
**.loRaNodes[2].**.initialY = 518.18m
**.loRaNodes[3].**.initialX = 484.83m
**.loRaNodes[3].**.initialY = 468.03m
**.loRaNodes[4].**.initialX = 547.28m
**.loRaNodes[4].**.initialY = 418.85m
**.loRaNodes[5].**.initialX = 442.3m
**.loRaNodes[5].**.initialY = 494.69m
**.loRaNodes[6].**.initialX = 528.67m
**.loRaNodes[6].**.initialY = 541.73m
**.loRaNodes[7].**.initialX = 551.7m
**.loRaNodes[7].**.initialY = 515.53m
**.loRaNodes[8].**.initialX = 425.33m
**.loRaNodes[8].**.initialY = 580.96m
**.loRaNodes[9].**.initialX = 501.35m
**.loRaNodes[9].**.initialY = 589.68m
[[432.51, 481.56], [545.87, 435.16], [476.36, 518.18], [484.83, 468.03], [547.28, 418.85], [442.3, 494.69], [528.67, 541.73], [551.7, 515.53], [425.33, 580.96], [501.35, 589.68]]
[69.96380278401111, 79.42469704065604, 29.82217295905846, 35.386576551003095, 93.9186930275331, 57.9438184796273, 50.62965336638203, 53.982135007796835, 110.1368716643069, 89.6901605528722]